service/src/main/java/uk/gov/hmcts/reform/fpl/validation/groups/ValidateCaseNumberGroup.java

```diff
@@ -1,5 +1,5 @@
 package uk.gov.hmcts.reform.fpl.validation.groups;
 
 // JSR Validation interface used to validate the C21 Group
```

Comment:
Update this comment please

Reply:
has this been updated @joez-f ?

Reply:
@joez-f  forgive me i thought i had pulled your latest from the branch :)

Reply:
No problem, yes I updated it :)